### ISSUE TITLE
Components / Button - Aria label missing

### ISSUE THREAD

**Issue by yesenia-gtz** [Office-of-Digital-Services/California-State-Web-Template-Website#972]
[Components / Button](https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/components/buttons.html)

btn-row p-b. color contrast of the ‘disabled buttons’ maybe difficult to see. [ARIA14: Using aria-label to provide an invisible label where a visible label cannot be used](https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html)

- [ ] Add an aria label for a good description. OR add a visible text description of the disabled buttons.

![image](https://user-images.githubusercontent.com/98193284/207443971-7736da14-ef7d-465c-ab69-d096069e14d3.png)

**Comment by kkoryaka** [comment]
@pglomboy this is warning. We are taking care of a11y issues. And there are no issues with this page.

**Comment by pglomboy** [comment]
Yes, this page is error free - I'm aware. Just making recommendations of adding an aria label as part of my manual testing. but we can move this to the complete bucket. @kkoryaka

**Comment by yesenia-gtz** [comment]
Woop woop! Thanks for the clarification! closing :)
